Source/bmalloc/bmalloc/Mutex.cpp

```diff
@@ -40,6 +40,14 @@ static inline void yield()
 #if BOS(DARWIN)
     constexpr mach_msg_timeout_t timeoutInMS = 1;
     thread_switch(MACH_PORT_NULL, SWITCH_OPTION_DEPRESS, timeoutInMS);
+#elif BPLATFORM(WPE)
+
+#ifdef WEBKIT_WPE_BMALLOC_MICROSECONDS_SLEEP
```

Comment:
You cound conflate these two preprocessor checks into just one, and avoid repeating the `sched_yield()` call:

```suggestion
#elif BPLATFORM(WPE) && defined(WEBKIT_WPE_BMALLOC_MICROSECONDS_SLEEP)
```

Reply:
That was a good suggestion, committed.